packages/mrkaplan/PKGBUILD

```
@@ -0,0 +1,21 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=mrkaplan
+pkgver=1.1.1
+pkgrel=1
+pkgdesc='Help red teamers to stay hidden by clearing evidence of execution.'
+arch=('any')
+groups=('blackarch' 'blackarch-windows' 'blackarch-exploitation' 'blackarch-recon')
```

```suggestion
groups=('blackarch' 'blackarch-windows' 'blackarch-exploitation')
```

not related to recon

---

> 

Since it is a "moonwalk fork" but for Windows, since Moonwalk reports in its README that one of the features is the [Reconnaissance](https://github.com/mufeedvh/moonwalk), is it correct or not to provide also `blackarch-recon` group?

---

We don't have to trust all what upstream says, plus the word is used in a different meaning

---

Removed `blackarch-recon`.